coordinator_query.hh

```diff
@@ -18,14 +18,12 @@
 
 namespace service {
 class client_state;
-class storage_proxy;
 };
-using clock_type = lowres_clock;
```

Comment:
Well, it was not pointless when it was in the storage_proxy. It made sure that everything related to sp uses the same clock. Now we just need to trust that we did not made a mistake.